FpsCheck.py

```diff
@@ -365,9 +365,11 @@ def find_invalid_well_known(self, check_sets):
             # have stored
             try:
                 json_schema = self.open_and_load_json(url)
-                schema_fields = set(self.acceptable_fields) & set(
-                    json_schema.keys())
                 curr_fps_set = check_sets[primary]
+                schema_fields = set({field for field in json_schema.keys()
+                                     if field in self.acceptable_fields})
```

Comment:
FYI: since `self.acceptable_fields` is always the same for every FpsCheck instance, and it is never modified, it really should be a file-level constant (i.e. `ACCEPTABLE_FIELDS`), instead of a class member variable.